ecs-agent/acs/session/attach_resource_responder.go

```diff
@@ -202,6 +202,10 @@ func validateAttachmentAndReturnProperties(message *ecsacs.ConfirmAttachmentMess
 		if err != nil {
 			return nil, errors.Wrap(err, "resource attachment validation by attachment type failed")
 		}
+		err = resource.ValidateFileSystemType(attachmentProperties[resource.FileSystemKey])
+		if err != nil {
+			return nil, errors.Wrap(err, "resource attachment validation by attachment type failed")
```

Comment:
error msg -> `resource attachment validation by the file system property failed` might be better IMO.